app/assets/javascripts/hyrax/i18n_helper.es6

```diff
@@ -0,0 +1,17 @@
+export default class InternationalizationHelper {
+  constructor() {
+    this.addLangClickListener();
+  }
+
+  addLangClickListener() {
```

Comment:
Can you add documentation that explains what this method is doing?

Reply:
Yea no problem.  I just updated the file.  This function handles the event of changing (by clicking), a new language in the top bar language selector dropdown.  And the thought was the class `InternationalizationHelper.js` could serve as a generic, helper class to organize internationalization event handling.